src/table/src/EditableCellField.js

```diff
@@ -112,9 +123,21 @@ const EditableCellField = memo(function EditableCellField(props) {
       default:
         break
     }
-  }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
 
-  const { minHeight = 40, minWidth = 80, size, value, zIndex } = props
+  const style = useMemo(
+    () => ({
+      left,
+      top,
+      height,
+      minHeight: Math.max(height, minHeight),
+      width,
+      minWidth: Math.max(width, minWidth),
+      zIndex
+    }),
+    [left, top, height, width, minHeight, minWidth, zIndex]
```

Comment:
I think we may want to consider ripping out some of this measurement/dimension logic in favor of a simple hook. There's a lot of complex logic that has nothing to do with this component in particular.

Reply:
200% measurement should definitely be a utility hook